packages/cli/src/commands/pg/promote.ts

```diff
@@ -0,0 +1,176 @@
+      const {body: backup} = await this.heroku.post<Heroku.AddOnAttachment>('/addon-attachments', {
+        body: {
+          app: {name: app},
+          addon: {name: current.addon?.name},
+          namespace: current.namespace,
+          confirm: app,
+        },
+      })
+      ux.action.stop(color.green(backup.name + '_URL'))
+    }
+
+    if (!force) {
+      const {body: status} = await this.heroku.get<PgStatus>(`/client/v11/databases/${attachment.addon.id}/wait_status`, {
+        hostname: pgHost(),
+      })
+      if (status['waiting?']) {
+        ux.error(heredoc(`
+          Database cannot be promoted while in state: ${status.message}
+          
+          Promoting this database can lead to application errors and outage. Please run ${color.cmd('pg:wait')} to wait for database to become available.
+          
+          To ignore this error, you can pass the --force flag to promote the database and risk application issues.
+        `))
+      }
+    }
+
+    let promotionMessage
+    if (attachment.namespace) {
+      promotionMessage = `Promoting ${color.cyan(attachment.name)} to ${color.green('DATABASE_URL')} on ${color.app(app)}`
+    } else {
+      promotionMessage = `Promoting ${color.addon(attachment.addon.name)} to ${color.green('DATABASE_URL')} on ${color.app(app)}`
+    }
+
+    ux.action.start(promotionMessage)
+    await this.heroku.post('/addon-attachments', {
+      body: {
+        name: 'DATABASE',
+        app: {name: app},
+        addon: {name: attachment.addon.name},
+        namespace: attachment.namespace || null,
+        confirm: app,
+      },
+    })
+    ux.action.stop()
+    const currentPooler = attachments.find(a => a.namespace === 'connection-pooling:default' && a.addon?.id === current.addon?.id && a.name === 'DATABASE_CONNECTION_POOL')
+    if (currentPooler) {
+      ux.action.start('Reattaching pooler to new leader')
+      await this.heroku.post('/addon-attachments', {
+        body: {
+          name: currentPooler.name,
+          app: {name: app},
+          addon: {name: attachment.addon.name},
+          namespace: 'connection-pooling:default',
+          confirm: app,
+        },
+      })
+      ux.action.stop()
+    }
+
+    const {body: promotedDatabaseDetails} = await this.heroku.get<PgDatabase>(`/client/v11/databases/${attachment.addon.id}`, {
+      hostname: pgHost(),
+    })
+    if (promotedDatabaseDetails.following) {
+      const unfollowLeaderCmd = `heroku pg:unfollow ${attachment.addon.name}`
+      ux.warn(heredoc(`
+        Your database has been promoted but it is currently a follower database in read-only mode.
+        
+        Promoting a database with ${color.cmd('heroku pg:promote')} doesn't automatically unfollow its leader.
+        
+        Use ${color.cyan(unfollowLeaderCmd)} to stop this follower from replicating from its leader (${color.yellow(promotedDatabaseDetails.leader as string)}) and convert it into a writable database.
```

Comment:
Maybe replace `color.cyan` with `color.cmd` here?

```suggestion
        Use ${color.cmd(unfollowLeaderCmd)} to stop this follower from replicating from its leader (${color.yellow(promotedDatabaseDetails.leader as string)}) and convert it into a writable database.
```